package.json

```diff
@@ -82,7 +81,6 @@
     "eslint-plugin-jest": "^27.0.1",
     "eslint-plugin-jsdoc": "^39.3.6",
     "eslint-plugin-jsx-a11y": "^6.6.1",
-    "eslint-plugin-prettier": "^4.2.1",
```

Comment:
Per Pretter this is:

> generally not recommended

And I don't think we fall under [the exceptions](https://prettier.io/docs/en/integrating-with-linters.html)